app/allocation/views/confirmation.njk

```diff
@@ -28,15 +29,9 @@
       </p>
 
       <p>
-        <a href="{{ ALLOCATIONS_URL }}">
+        <a href="/allocations">
           {{ t("allocations::confirmation.view_all") }}
         </a>
-        <span> {{t("or")}} </span>
-        <a href="{{ ALLOCATIONS_URL }}">
-          {{ t("allocations::confirmation.view_by_from_location", {
```

Comment:
Could also remove this one too.

Reply:
Both should go back in soon, mind. The removal is meant to be temporary.